src/renderer/gl_engine/tvgGlRenderTask.cpp

```diff
@@ -111,11 +111,10 @@ void GlComposeTask::run()
     GL_CHECK(glBindFramebuffer(GL_FRAMEBUFFER, getSelfFbo()));
 
     // clear this fbo
-    GLenum color_buffer = GL_COLOR_ATTACHMENT0;
-    const float transparent[] = {0.f, 0.f, 0.f, 0.f};
-
-    GL_CHECK(glDrawBuffers(1, &color_buffer));
```

Comment:
@RuiwenTang  removed this color buffer.

Reply:
If examples/Animation can be rendered correctly, I think it can be removed.

Reply:
Sure, it works.